tests/test_detect_target.py

```diff
@@ -11,6 +11,8 @@
 from modules.detect_target import detect_target
 from modules import frame_and_time
 
+if not torch.cuda.is_available():
```

Comment:
Why?

Reply:
Test cases keep failing on workflow because of a PyTorch error. Added this because I thought it might be related to CUDA not being available.

Reply:
You can see the error in the failed builds.